apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -331,12 +338,16 @@ defmodule OMG.Watcher.ExitProcessor do
     _ = if not Enum.empty?(events), do: Logger.info("Recognized in-flight exits: #{inspect(events)}")
 
     contract_ife_ids =
-      Enum.map(
-        events,
+      events
+      |> Task.async_stream(
```

Comment:
`ExPlasma.InFlightExit.txbytes_to_id(txbytes)` is already a local computation though, no longer making RPC calls. 

Maybe at least we can remove the custom timeout and so defaults back to 5000ms which is plenty enough?

Reply:
oh geez. I didn't even check. Now I remember you did this!